examples/pcr/extend.c

```diff
@@ -0,0 +1,201 @@
+/* extend.c
+ *
+ * Copyright (C) 2006-2020 wolfSSL Inc.
+ *
+ * This file is part of wolfTPM.
+ *
+ * wolfTPM is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * wolfTPM is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301, USA
+ */
+
+/* This is a helper tool for extending hash into a TPM2.0 PCR */
+
+#include <wolftpm/tpm2_wrap.h>
+#include <wolfssl/wolfcrypt/hash.h>
+
+#include <examples/pcr/extend.h>
+#include <examples/tpm_io.h>
+#include <examples/tpm_test.h>
+
+#include <stdio.h>
+
+
+/******************************************************************************/
+/* --- BEGIN TPM2.0 PCR Extend example tool  -- */
+/******************************************************************************/
+
+inline static void usage(void)
+{
+    printf("Expected usage:\n");
+    printf("extend (pcr_number) (filename)\n");
```

Comment:
Better usage template: `./examples/pcr/extend [pcr_number] [filename]`. Also add `(default 16) in the `pcr_number` description.